cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -14,18 +14,25 @@ if [ -n "${KUBEVIRTCI_TAG}" ] && [ -n "${KUBEVIRTCI_GOCLI_CONTAINER}" ]; then
     echo "WARNING: KUBEVIRTCI_GOCLI_CONTAINER is set and will take precedence over the also set KUBEVIRTCI_TAG"
 fi
 
-if [ "${KUBEVIRTCI_RUNTIME}" = "podman" ]; then
-    _cli="pack8s"
+if docker ps >/dev/null; then
+    _cri_bin=docker
+    echo "selecting docker as container runtime"
+elif podman ps >/dev/null; then
+    _cri_bin=podman
```

Comment:
What do you think about doing some sanity checks here with the stuff documented, like podman version, rest socket checks, since it's already at the documentation is not a must.

Reply:
That is a good idea.

Reply:
Done.